.env.template

```
@@ -0,0 +1,9 @@
+# Copy this file to .env and fill in your actual values
+# .env is already in .gitignore so it won't be committed
+
+# Smithery API Configuration
+SMITHERY_API_KEY=your-smithery-api-key-here
+SMITHERY_PROFILE=your-smithery-profile-here
+
+# QuickNav Configuration
+QUICKNAV_TRAINING_DIR=C:/Users/SamLyndon/.quicknav/training_data
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)